container_meals/models/res_partner.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 Coop IT Easy SCRL fs
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    total_container_price = fields.Float(
+        string="Total amount spent on containers",
+        digits="Product Price",
+        compute="_compute_containers_deposits",
+    )
+    total_deposit_price = fields.Float(
+        string="Total discount received from deposit products",
+        digits="Product Price",
+        compute="_compute_containers_deposits",
+    )
+    current_deposit = fields.Float(
+        string="Current Container Deposit",
+        digits="Product Price",
+        compute="_compute_containers_deposits",
+    )
+    container_order_line_ids = fields.One2many(
```

Comment:
as i understand, this corresponds to the order lines linked to containers or to the deposit product. wouldn’t it be more logical to have 2 separate properties?

Reply:
This should probably be renamed. Having 2 separate properties mightn't work, though. I'm not sure if two One2many fields can be combined into a single table for the user.

Reply:
Renamed this field to `deposit_related_order_line_ids`.